src/google-recaptcha-provider.tsx

```diff
@@ -28,12 +28,25 @@ interface IGoogleReCaptchaProviderProps {
     async?: boolean;
     appendTo?: 'head' | 'body';
     id?: string;
+    onLoadCallbackName?: string;
```

Comment:
Are you sure this callback is needed? If the user specifies it in the parameters, then the logic with the inline badge will break